src/device-vibration.cc

```diff
@@ -423,12 +421,10 @@ void VibrationSettingsWidget::sendVibrateCommand()
   // Spotlight:
   //                                                    len         intensity
   // unsigned char vibrate[] = {0x10, 0x01, 0x09, 0x1a, 0x00, 0xe8, 0x80};
+
   const uint8_t vlen = m_sbLength->value();
   const uint8_t vint = m_sbIntensity->value();
   const uint8_t vibrateCmd[] = {0x10, 0x01, 0x09, 0x1a, vlen, 0xe8, vint};
 
-  const auto res = m_subDeviceConnection->sendData(vibrateCmd, sizeof(vibrateCmd));
-  if (res != sizeof(vibrateCmd)) {
-    logWarn(device) << "Could not write vibrate command to device socket.";
-  }
```

Comment:
Why remove the warning output?

Reply:
This warning is removed because it is not exhaustive and better alternate now exist in the modified code. Vibration command can fail in two ways:

1. Device not active: This warning do not check for this case. However, proper check is now provided in `sendData` function (src/device.cc L372).
2. Failed to write to device file: This case is checked in `sendData` function and proper warning is issued (src/device.cc L386).

Additionally, there was no mean to check that device actually vibrated (or not); however in the current code check the vibration acknowledgement packet and confirm device vibration (src/spotlight.cc L332).